### DISCUSSION TITLE
Design ideas

### DISCUSSION THREAD

**Discussion by bonomat** [itchysats/10101#273] [General]
Below are some ideas, not all of them have to be actioned now or exactly how to proposed.

**Comment by bonomat** [comment]
# Pre-start


Add splash screen / loading screen

**Comment by bonomat** [comment]
# Them color

Change orange to something _softer_. Maybe blue?

**Comment by bonomat** [comment]
# Dashboard

![simulator_screenshot_EBE43B7A-01D8-43D6-9715-CF90C5E529FE](https://user-images.githubusercontent.com/224613/203005893-222987eb-1e08-44ff-849d-2a03f4a4b475.png)

## Balance

- Play around with the balances. Stacking Lightning on top Bitcoin balance can be nicer.
- The Bitcoin logo next to the on-chain balance can be confusing: Is it the denominator or just indicating if it is on-chain or off-chain.

## Separators

Look a bit harsh. Maybe we can make them a bit subtle. 

## Cfd Trading and Sports Betting buttons

- Remove shadows and add a thin border color inline with theme color

## `Create Wallet Backup` and `Deposit Bitcoin`

- Have no shadows. Either add or remove shadows from App buttons above.
- Left line of alerts could need some color, maybe orange? 
- Make soft corners
- Vertically and horizontally align the icon on the left
- Justify the text

**Reply by da-kami** [reply]
> The Bitcoin logo next to the on-chain balance can be confusing: Is it the denominator or just indicating if it is on-chain or off-chain.

Change into chain icon.

**Reply by da-kami** [reply]
Separator: Align with theme color, thinner

**Comment by bonomat** [comment]
# Backup Seed

![simulator_screenshot_62FA80D2-9328-40B3-81EB-D55E076C9A1B](https://user-images.githubusercontent.com/224613/203010817-1e427a0b-dc1d-4089-83d8-741b541b4478.png)

- Change the wording, it was a c&p from a different wallet
- Fix lengths of visible and hidden seed. It's jumping when clicking on show/hide seed

**Comment by bonomat** [comment]
# Receive Bitcoin

![simulator_screenshot_BF00C236-64C5-4B99-8B58-5A2A7680287C](https://user-images.githubusercontent.com/224613/203012342-29159d18-bd0d-4447-969a-8e15ea05274b.png)

- Make it possible to see the whole address
- `Copy Address` should close the `Receive Bitcoin` screen. Maybe rename it to close?
- Add a 10101 logo into the QR code

**Comment by bonomat** [comment]
# Dashboard after receiving (pending transaction)

![simulator_screenshot_577F35E6-BD4B-499F-85FE-0C30AF92A5D1](https://user-images.githubusercontent.com/224613/203013348-32476f1f-0807-4c54-b425-280b3f31ecf8.png)

- `Bitcoin Received` should be online. Maybe change the text? 
- Every information should be on the same _level_, i.e. the tx history is on the lowest level and the `CFD Trading` on the highest which even has a shadow. 
- Add a date, when this transaction was confirmed

Potentially get inspired by Zeus: 
![image](https://user-images.githubusercontent.com/224613/203014401-cb681a2e-b72b-4148-82b6-5b9987cb93ed.png)

**Reply by da-kami** [reply]
* add time indicator (second row)
* Ellipse (overflow) on Text in the middle
* Reduce padding (in list as well as outer)

Service tiles:

* Remove shade, outline with theme color (? - see how that looks)

**Comment by bonomat** [comment]
# Open Channel

![simulator_screenshot_2DE4292F-F115-457B-89EC-4B2A14E19EE9](https://user-images.githubusercontent.com/224613/203014991-1d557bec-c017-4b91-b49f-4afd18230574.png)

- The user is flooded with information.
- The screen could use some icons (e.g. behind the balances)
- Make the p2p endpoint less important but show the most important information more prominent, i.e. the amount you want to lock up. 
- Node name: this can be omitted. 
- Is there a reason why this screen is not consistent with the other screen, i.e. have the balance at the top?
- Idea: change the text to something like: `We only support a single channel with 10101. More advanced usecases will come later`. 
- Add icons to explain what is happening, e.g. something like this: 
![image](https://user-images.githubusercontent.com/224613/203016714-e7f7a64d-732a-494f-983d-a7ca27fe694e.png)
- Opening a channel should redirect to a success page showing some information, e.g. a tx link to mempool and text saying: `You successfully initiated the channel creation. You will be able to use it once the funding transaction is confirmed`
- In the failure case we should display an error

**Reply by da-kami** [reply]
> Is there a reason why this screen is not consistent with the other screen, i.e. have the balance at the top?

Yes. It is a child screen showing details of an action. So far we did not display the balance on child/detail screens but only on the outer screens. We can change that though, it's not a hard rule.

> Idea: change the text to something like: We only support a single channel with 10101. More advanced usecases will come later.

We can add that, but I think it depends on what we want to showcase: What is possible vs. what is currently implemented. If we showcase what will be possible here then we don't need changes, if it is important to show that there is a current limitation (that we might not be able to overcome easily) we can adapt the text :)

**Reply by da-kami** [reply]
> Add icons to explain what is happening, e.g. something like this:

I have a hard time imagining how you would achieve this for this screen. Do you mean something like steps with icons that show what will happen? Or one icon (at the top) that indicates the channel open?

**Reply by da-kami** [reply]
* align the design with what was designed in the wireframe, but:
    * remove node name, 

* channel amount has to be most present
* if time: Add icon and redice text; icon on top, text under, then the channel amount

**Comment by bonomat** [comment]
# Icons

Use Bitcoin Icons, ie for Satoshi Icon 

https://bitcoinicons.com/

![simulator_screenshot_74AC3031-9C23-4597-B206-FFD4AA3B9573](https://user-images.githubusercontent.com/224613/203005848-19246dd9-325c-4c89-b84b-91be10bd8e8c.png)

**Comment by holzeis** [comment]
## Buttons

Our buttons are quite small - I think it would improve the look and feel significantly if we design a bigger / nicer button.

**Reply by da-kami** [reply]
I think this problem becomes more noticeable on bigger screens; on the iPhone SE I never felt the buttons were overly small. Eventually we might want to invest some time into learning [how to build better responsive / adaptive apps](https://docs.flutter.dev/development/ui/layout/adaptive-responsive).

I was wondering it it would be a good idea to stick to FABs and use the more universally, i.e. also in our child screens (e.g. confirmation button). But I'm not sure that would help.
